terasoluna-gfw-parent/pom.xml

```diff
@@ -398,7 +397,7 @@
     <org.mybatis.version>3.4.6</org.mybatis.version>
     <org.mybatis.spring.version>1.3.2</org.mybatis.spring.version>
     <!-- == Dozer== -->
-    <dozer.version>6.3.0</dozer.version>
+    <dozer.version>6.4.0</dozer.version>
```

Comment:
@btshuugna 
- [x] I checked them.
* [6.4 release note](https://github.com/DozerMapper/dozer/releases/tag/6.4.0)
* [6.3 to 6.4 migration guide](https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/migration/v63-to-v64.adoc)

And understand it's no problem to upgrade.
But it seems to need reflect follows topic to our guideline.
* Java 8 date and time support